### PR TITLE
Revert PR #11 and pick up changes from PR #15

### PR DESCRIPTION
**Pull request by swarajpure**
image

### CODE REVIEW

**Review by whyDontI — APPROVED** [review]
LGTM 👍🏽

File: main.css [hunk 1]
```diff
@@ -5,72 +5,65 @@
...
+  padding: 1em 0;
 }
 
-.container .inputBox {
-  position: relative;
-  padding: 2px;
+#taskForm {
+  width: 60vw;
+  max-width: 40rem;
+  min-width: 300px;
```
Comment (akankshadharkar) [inline main.css:22]: Are these to ensure height when the screen size changes?

Comment (swarajpure) [reply main.css:22]: @kamara-moses please check this comment

Comment (kamara-moses) [reply main.css:22]: @akankshadharkar yes that is to ensure the height when the screen size changes.

**Pull request merged**